src/joint.dia.element.js

```diff
@@ -694,8 +694,6 @@ joint.dia.ElementView = joint.dia.CellView.extend({
 
     pointerdown: function(evt, x, y) {
 
-        this.model.trigger('batch:start');
-
         // target is a valid magnet start linking
         if (evt.target.getAttribute('magnet') && this.paper.options.validateMagnet.call(this.paper, this, evt.target)) {
 
```

Comment:
The batch:start is getting called in CellView.prototype.pointerdown, duing both Element:pointerdown and Link:pointerown; adding it back here causes a extra unnecessary bactch:start;  For adding a link it becomes Element:batch:start- Link:batch:start; but since you are really just passing the pointerdown to the link handers on magnet click, it seems unnecessary;

Since the batch:start/stop is necessary in the CellView prototype due to having to deal with those events on delete, in my opinion they are not necessary in the derived.  In this implmentation we are relying on the CellView prototype to handle the batch start/stop, which is why they arent present in LinkView or CellView pointer down/up